canonical-websites/www.ubuntu.com
# Remove news page from Kubernetes docs

## Done

remove templates/kubernetes/docs/news.md
Adjust navigation in templates/kubernetes/docs/shared/_side-navigation.md

The news page doesn't really belong in documentation and won't be updated

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- check that the news page is no longer listed in the navigation
